translations/tiled_de.ts

```diff
@@ -593,17 +593,17 @@
     <message>
         <location line="+8"/>
         <source>&amp;Never</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Nie</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>For &amp;Selected Objects</source>
-        <translation type="unfinished"></translation>
+        <translation>Für ausge&amp;wählte Objekte</translation>
```

Comment:
Right, for these accelerators, you should adjust them to the language and just avoid conflicts within the same menu. You guys decide whether **s** or **w** is best here. :-)